src/plugins/intel_gpu/src/graph/program_node.cpp

```diff
@@ -1505,7 +1505,11 @@ void program_node::create_onednn_primitive_attributes(
                     oc_dim = static_cast<int>(desc.output_layout.get_partial_shape()[1].get_max_length());
                 else
                     oc_dim = static_cast<int>(desc.output_layout.get_tensor().feature.size());
-                post_ops.append_prelu(1 << std::max(0, oc_dim));
+                if (oc_dim >= 0) {
+                    post_ops.append_prelu(1 << oc_dim);
+                } else {
+                    throw std::runtime_error("oc_dim is negative, which is not allowed for bit shift operation.");
+                }
```

Comment:
Id better report as false positive, max from 0 and value can not be negative

Reply:
Updated Coverity status.

Reply:
Can be resolved by cast to unsigned std::max(0, oc_dim)